js/ad_manager_vast.js

```diff
@@ -640,6 +884,52 @@ OO.Ads.manager(function(_, $) {
     };
 
     /**
+    * This method pings all the ad's error URLs with a specific error code if the error URL
+    * contains the macro, "[ERRORCODE]". If ad has a parent wrapper, then go up the chain and ping
+    * wrapper's error urls as well.
+    * @public
+    * @method Vast#trackError
+    * @param {number} code Error code
+    * @param {boolean} currentAdId Ad ID of current ad
+    */
+    this.trackError = _.bind(function trackErrorHelper(code, currentAdId) {
+      if (currentAdId && currentAdId in this.errorInfo) {
+        pingURLs(this.errorInfo[currentAdId].errorUrls);
+        var parentId = this.errorInfo[currentAdId].wrapperParentId;
+
+        // ping parent wrapper's error urls too if ad had parent
+        if (parentId) {
+          this.trackErrorHelper(code, parentId);
```

Comment:
I see - i think the name only matters if you plan on assigning it to another function and changing it.  Since this function won't change or be mapped to another function name i think it's okay to stay with one name